tools/database_files_update.pl

```diff
@@ -18,7 +18,7 @@ =head1 SYNOPSIS
 =head1 DESCRIPTION
 
 This script updates the path stored in the C<files> and C<parameter_file>
-tables to remove the <\$data_dir> part of the path for security improvements.
+tables to remove the C<$data_dir> part of the path for security improvements.
```

Comment:
just a tiny tiny comment (data_dir on this line and 10 lines earlier is inconsistent; see screenshot)... I don't mind either way, but it might be good to make them look the same!
![data_dir](https://user-images.githubusercontent.com/15198379/38516253-a325fc6a-3c04-11e8-8dfa-876b1754a664.png)